channels.yaml

```diff
@@ -201,5 +211,12 @@ releases:
   - version: v1.23.6+k3s1
     minChannelServerVersion: v2.6.4-alpha1
     maxChannelServerVersion: v2.6.99
-    serverArgs: *serverArgs-v2
+    agentArgs: *agentArgs-v2
```

Comment:
Ahhh wait, it looks like serverargs got removed here by accident?

Reply:
Good catch!